src/main/java/com/ibm/infrastructure/database/readmodels/CBOMReadRepository.java

```diff
@@ -75,7 +76,7 @@ public CBOMReadRepository(@Nonnull IDomainEventBus domainEventBus) {
     }
 
     @Override
-    public @Nonnull Optional<CBOMReadModel> findBy(@Nonnull GitUrl gitUrl) {
+    public @Nonnull Optional<CBOMReadModel> findBy(@Nonnull ScanUrl gitUrl) {
```

Comment:
```suggestion
    public @Nonnull Optional<CBOMReadModel> findBy(@Nonnull Purl purl) {
```
should be "findeble" by gitUrl or PURL